### PR TITLE
Create Space Travelers Page

### PR DESCRIPTION
**Pull request by Javivy**
## In this project my teammate and I implemented the following functions and enhancements:

  - [x] Created two different pages for rockets and missions
  - [x] Fetched API for rockets and missions
  - [x] Made use of the API to display the correspondent data for every element
  - [x] Styled each page with CSS modules
  - [x] Work as a team
  - [x] Meet deadlines

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
# Hi Team, 👋🏻👋🏻 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you don't have a README file customized so kindly customize it to your information and submit the PR again 👋🏻🚀🔥💯

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Components/Rockets/RocketItem.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { reserveRocket } from '../../Redux/Rockets';
+import css from './RocketItem.module.css';
+
+// # needed variable from rockets.rockets
+/*
+  rocket_name
+  active
+  description
+  flickr_images
+*/
```
Comment (Reem-lab) [inline src/Components/Rockets/RocketItem.jsx:13]: - [ ] I recommend removing the commented code here from this file to make you code clean and dry ✨🔥🚀💯

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻

Good job so far! 🔥🚀🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✨✨
- No Linters error 🔥🔥
- Followed Gitflow 🔥🔥
- Good commit messages 🔥🔥
- Great job on both the design and the functionality 🔥🔥
- Professional README file 🔥🔥🔥


## Required Changes ♻️
_Check the comments under the review._
- [ ] Greay job but I think you forget to make the tests for your components so kindly make tests to your pages using Jest library so kindly according to the requirements make the tests 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi team,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 
approved


## Highlights
- Your project works as expected 👍 👍 
- All unit tests are passing ✔️ 
- All linter checks are passing ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**